src/sentry/utils/snowflake.py

```diff
@@ -28,7 +29,8 @@ def save_with_snowflake_id(self, snowflake_redis_key, save_callback):
                 return
             except IntegrityError:
                 self.id = None
-        metrics.incr("snowflake_id_creation_failed", sample_rate=1)
+
+        capture_exception(MaxSnowflakeRetryError)
```

Comment:
```suggestion
        capture_exception(MaxSnowflakeRetryError())
```

I think you need an instance not a class.